client/src/components/Bookings/BookingCard.tsx

```diff
@@ -0,0 +1,46 @@
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import useStyles from './useStyles';
+import AvatarDisplay from '../AvatarDisplay/AvatarDisplay';
+import { Booking } from '../../interface/Booking';
+import moment from 'moment';
+import BookingMenu from './BookingMenu';
+
+interface Props {
+  booking: Booking;
+  component?: string;
```

Comment:
Th property name `component` seems a bit misleading, why don't we rename it to `type` instead?

Reply:
Thanks @rajivtitus , I will do that.